lib/multi_password/strategies/argon2.rb

```diff
@@ -9,12 +9,29 @@ class Argon2
       register :argon2
 
       def create(password, options = {})
-        ::Argon2::Password.new(options).create(password)
+        ::Argon2::Password.new(validate_options(options)).create(password)
       end
 
       def verify(password, encrypted_password)
         ::Argon2::Password.verify_password(password, encrypted_password)
       end
+
+      def validate_options(options)
+        return options if options.empty?
+
+        t_cost = options[:t_cost]
+        m_cost = options[:m_cost]
+
+        if !t_cost.is_a?(Integer) || t_cost < 1 || t_cost > 750
+          raise InvalidOptions.new('argon2', 't_cost must be an integer between 1 and 750')
+        end
+
+        if !m_cost.is_a?(Integer) || m_cost < 1 || m_cost > 31
+          raise InvalidOptions.new('argon2', 'm_cost must be an integer between 1 and 31')
```

Comment:
Could we make these options optional rather than required? For example, user should be able to define options as something like this:
```ruby
config.default_options = { t_cost: 1 }
```